docs/src/Dropdown.doc.js

```diff
@@ -368,57 +801,42 @@ function SectionsIconButtonDropdownExample() {
         accessibilityHaspopup
         accessibilityExpanded={open}
         accessibilityLabel="More Options"
-        icon="arrow-down"
+        icon="add"
+        iconColor="darkGray"
```

Comment:
this button looks so dark , but not sure if it's ok or not
![image](https://user-images.githubusercontent.com/10593890/112027309-71804700-8b0d-11eb-919a-9d7a20467bee.png)